crates/ere-hosts/src/main.rs

```diff
@@ -194,7 +194,7 @@ fn get_zkvm_instances(
     {
         #[cfg(feature = "sp1")]
         {
-            run_cargo_patch_command("sp1")?;
+            run_cargo_patch_command("sp1", Some(guest_program_folder))?;
```

Comment:
`run_cargo_patch_command` now accepts an optional guest program dir where you want to do the crate patching.